src/changes/changes.xml

```diff
@@ -72,6 +72,22 @@ https://maven.apache.org/plugins/maven-changes-plugin/xsd/changes-1.0.0.xsd
     </release>
     -->
     <release version="0.17-SNAPSHOT" date="xxxx-yy-zz" description="Current SNAPSHOT - release to be done">
+      <action issue="RAT-54" type="fix" dev="claudenw">
+        MIME Detection Using Tika
+      </action>
+      <action issue="RAT-20" type="fix" dev="claudenw">
+        Changed to detecting binary by content not name.
+      </action>
+      <action issue="RAT-147" type="fix" dev="claudenw">
+        Change to detect non UTF-8 text as text not binary.
```

Comment:
Isn't that what RAT-301 is all about? I got curious when I read your comment in the changelog here.

Reply:
https://github.com/apache/linkis/blob/master/linkis-public-enhancements/linkis-pes-common/src/main/java/org/apache/linkis/udf/entity/UDFVersion.java

Reply:
@Claudenw would you mind integrating above file in order to see if its properly detected with Tika?

Reply:
@Claudenw I brought in changes related to RAT-301 - is that fine for you or should the Chinese character example go somewhere else? Thanks